src/test/java/org/opensearch/ad/ml/CheckpointDaoTests.java

```diff
@@ -1000,4 +1003,14 @@ public void testFromEntityModelCheckpointWithEntity() throws Exception {
         }
         return point;
     }
+    // .parseReader(new FileReader(new File(getClass().getResource(labalFileName).toURI()), Charset.defaultCharset()))
+
+    public void testDeserializeRCFModel() throws Exception {
+        // Model in file 1_3_0_rcf_model.json not passed initialization yet
```

Comment:
So it means the model is empty? If yes, have you tested model fully initialized?